tests/iscsi/iscsi_client.pm

```diff
@@ -106,6 +106,7 @@ sub initiator_connected_targets_tab {
 sub run {
     prepare_xterm_and_setup_static_network(ip => $test_data->{initiator_conf}->{ip}, message => 'Configure MM network - client');
     zypper_call("in open-iscsi");
+    zypper_call("in yast2-iscsi-client");
```

Comment:
You can just install `yast2-iscsi-client` it will pull in `open-iscsi`.

Reply:
Ok, thanks, updated. I will run a new VR for it.

Reply:
this doesn't happen anymore for new products since YaST has containerized YaST modules, the deps needs to be installed explicitly.